src/main/java/htsjdk/samtools/GATKReadSparkCodec.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009 The Broad Institute
+ *
```

Comment:
You are reviewing dead code -- it's already removed in https://github.com/broadinstitute/gatk/pull/1127, which this branch is based off of. We should rebase this branch once https://github.com/broadinstitute/gatk/pull/1127 is merged to avoid confusion during review.